src/test/java/io/zold/api/RtTransactionTest.java

```diff
@@ -26,19 +26,29 @@
 import java.io.IOException;
 import nl.jqno.equalsverifier.EqualsVerifier;
 import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
 import org.hamcrest.core.IsEqual;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 /**
  * Test case for {@link RtTransaction}.
  *
  * @since 0.1
  * @checkstyle LineLengthCheck (500 lines)
  * @checkstyle JavadocMethodCheck (500 lines)
+ * @checkstyle MagicNumber (500 lines)
  */
 @SuppressWarnings({"PMD.AvoidCatchingGenericException", "PMD.TooManyMethods"})
 public final class RtTransactionTest {
 
+    /**
+     * Rule for checking expected thrown exceptions.
+     */
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
@carlosmiranda this should be marked as final, I'm surprised that qulice didn't complain…